src/app/dashboard/roles/page.tsx

```
@@ -0,0 +1,66 @@
+    setRoles((prev) => prev.filter((role) => role.id !== id));
+  };
+
+  return (
+    <div className="p-6">
+      <div className="flex justify-between items-center mb-6">
+        <h1 className="text-2xl font-bold text-gray-800">Manage Roles</h1>
+        <button
+          onClick={() => handleOpenModal()}
+          className="bg-amber-600 text-white px-4 py-2 rounded-lg hover:bg-amber-700"
+        >
+          <i className="fas fa-plus mr-2"></i>Add Role
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)